python/pyarrow/tests/test_extension_type.py

```diff
@@ -1319,38 +1319,65 @@ def test_tensor_type():
 
 
 def test_tensor_class_methods():
+    from numpy.lib.stride_tricks import as_strided
+
+    values = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+    np_arr = np.array(values, dtype=np.int8)
```

Comment:
The tensor is using float32?

Reply:
Indeed. I parameterized the test so these are no longer static.